package measure_cut_msgs.msgs;


message MeasureCut
{
	required bool initial_measure = 1;
	required bool cut = 2;
	required double action_bottom_x = 3;
	required bool lasagna_collision = 4;
}
